### PR TITLE
Improve alert styling

### PR DESCRIPTION
**Pull request by SamZhang02**
Improve alert styling
- Removed (X), clicking anywhere on the alert closes it 
- Improved color, it's now a bit less flashy to the eye

### CODE REVIEW

**Comment by SamZhang02** [comment]
yooo no way CI passed

File: client/src/components/Alert.tsx [hunk 1]
```diff
@@ -17,17 +12,24 @@ const defaultMessages = {
...
-  info: <AiOutlineInfoCircle className='text-blue-700 ' size={20} />,
-  warning: <IoWarningOutline className='text-yellow-700' size={20} />,
+  error: <IoWarning className='text-red-400 dark:opacity-80' size={25} />,
+  success: (
+    <AiFillCheckCircle className='text-green-400 dark:opacity-80' size={23} />
+  ),
+  info: (
+    <AiFillInfoCircle className='text-blue-300 dark:opacity-80' size={23} />
+  ),
+  warning: <IoWarning className='text-yellow-400 dark:opacity-80' size={25} />,
```
Comment (terror) [inline client/src/components/Alert.tsx:32]: I think we should have a different icon for `error` and `warning`. I suggest keeping the icon for warning but change the `error` icon to `VscError`

**Comment by terror** [comment]
We should use this instead 

https://sonner.emilkowal.ski/

**Comment by terror** [comment]
Closing in favour of #416

**Pull request closed without merging**